### ISSUE TITLE
QCoDeS can't register proxy parameters based on parameter_class

### ISSUE THREAD

**Issue by ttshaw1** [toolsforexperiments/instrumentserver#59]
My VNA, the Copper Mountain M5180, has qcodes parameters for each s-parameter, defined like
```
self.add_parameter(name='s11',
                           start=self.start(),
                           stop=self.stop(),
                           npts=self.npts(),
                           parameter_class=FrequencySweepMagPhase)
```
The instrument server and client don't have any problems with that parameter, but trying to add that the proxy parameter with `meas.register_parameter(vna.s21)` gives `ValueError: Can not register object of type <class 'method'>. Can only register a QCoDeS Parameter.`. It seems the client doesn't properly make that function a qcodes parameter when it's created.

**Comment by ttshaw1** [comment]
Ok, it turns out that `class FrequencySweepMagPhase(MultiParameter):` is a MultiParameter, which isn't yet supported in instrument server, so the server doesn't add s11 to the parameters blueprints. I'm going to see if I can get a handle on how hard it would be to add support for MultiParameters and ArrayParameters.

**Comment by wpfff** [comment]
@ttshaw1 yeah, i totally agree. we just didn't have any multiparameter instruments in use when we initially wrote this so we skipped dealing with that. would be absolutely worthwhile to have.
do you want to give this a shot? :)

**Comment by ttshaw1** [comment]
I've got it on my to-do list but don't know when I'll be able to spend the time on it. If anyone else scoops it up I certainly won't be offended

**Comment by ttshaw1** [comment]
@wpfff please see linked pull request above
